### ISSUE TITLE
Ender 3 MCU 'mcu' shutdown: Can't assign oid

### ISSUE THREAD

**Issue by se7enblade** [KevinOConnor/klipper#1806]
Installed Klipper on an Ender 3 with BLTouch but no matter what I do the configs fail, even from known working .cfg's and status brings up:

MCU 'mcu' shutdown: Can't assign oid

Worst part is I have had Klipper running before but abandoned as it was still early days with BLTouch/clones.
I've just spent four hours last night and another three this morning googling and trawling through reddit and github without success.
log file enclosed 

[klippy.log](https://github.com/KevinOConnor/klipper/files/3384719/klippy.log)

**Comment by klipper-gitissuebot** [comment]
Hi @se7enblade,

It did not look like there was a Klipper log file attached to this ticket. The log file has been engineered to answer common questions the Klipper developers have about the software and its environment (software version, hardware type, configuration, event timing, and hundreds of other questions).

Unfortunately, too many people have opened tickets without providing the log. That consumes developer time; time that would be better spent enhancing the software. If this ticket references an event that has occurred while running the software then the Klipper log must be attached to this ticket. Otherwise, this ticket will be automatically closed in a few days.

For information on obtaining the Klipper log file see: https://github.com/KevinOConnor/klipper/blob/master/docs/Contact.md

The log can still be attached to this ticket - just add a comment and attach the log to that comment.

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being.

**Comment by se7enblade** [comment]
[klippy.log](https://github.com/KevinOConnor/klipper/files/3384720/klippy.log)
To keep bot happy, apparently it's blind

**Comment by KevinOConnor** [comment]
Your micro-controller keeps resetting when the host tries to configure it.  Try updating to the latest very code from today (see the FAQ for upgrade details) and reflash the micro-controller.  Also, double check that you have an atmega1284p (and not some other type of avr chip).

-Kevin

**Comment by KevinOConnor** [comment]
I'm going to close this issue as it appears the conversation has concluded.

-Kevin

**Comment by Jufik** [comment]
I'm having the exact same issue.
I've been using Klipper with my EnderPro3, work(ed) like a charm.

I've received a [BLTouch kit](https://www.amazon.fr/gp/product/B07T4GFCVC/ref=ppx_yo_dt_b_asin_title_o03_s00?ie=UTF8&psc=1), and installed it.

Eventually, the kit comes with a USBISP drive, and I've flashed a bootloader. ([See instructions](https://www.reddit.com/r/ender3/comments/cfmbdy/howto_installing_a_bootloader_to_an_ender_3_pro/))
I've tried Official firmware provided by Creality, and it works fine.

Now, I just want to keep using Klipper, as it gives clear advantages.
*But*, since I've installed the BLTouch, Klipper behaves badly.

In the RPI side, I've redone the install procedure from scratch,
Without changing the stock Ender Config. 
Flashing happens smoothly, but boy, the result is odd.

The BLTouch kit comes with an  Pin 27 Board  to plug BLTouch in, and my guess is that it's what is messing up the config': The screen now is messy.

Removing the [display] from my config' seems make things a bit better, and my guess is there's some mix up on the pins', but boy... that's beyond my knowledge.
